chia/wallet/wallet_blockchain.py

```diff
@@ -226,3 +226,6 @@ async def clean_block_records(self) -> None:
 
         for header_hash in to_remove:
             del self._block_records[header_hash]
+
+    async def get_block_record_from_db(self, header_hash: bytes32) -> Optional[BlockRecord]:
```

Comment:
if you're adding this function, I think you want to add it to `BlockchainInterface` as well.

Reply:
oh, it seems it's already a part of `BlockchainInterface`. never mind